backend/utils/features.js

```diff
@@ -0,0 +1,13 @@
+const fs = require('fs');
+const path = require('path');
+
+const getComponentFeatureFlags = () => {
+  const normalizedPath = path.join(__dirname, '../../data/features.json');
+  try {
+    return JSON.parse(fs.readFileSync(normalizedPath, 'utf8'));
```

Comment:
future: since not sure if that may be a file prone to update, then using what you have is good, JSON can also be brought through with a `require('[your json file]')`

Reply:
the idea is that the feature flags can be enabled on the deployed site